netkan/netkan/status.py

```diff
@@ -79,50 +81,64 @@ def export_all_mods(cls, compat: bool = True) -> Dict[str, Any]:
         return data
 
     @classmethod
-    def export_to_s3(cls, bucket: str, key: str, compat: bool = True) -> None:
+    def export_all_mods(cls, compat: bool = True) -> Dict[str, Any]:
+        data: Dict[str, Any] = {}
+        for game_id in ['ksp', 'ksp2']:
```

Comment:
Is there any way we could get the list of game ids from the command line or the db, instead of hard-coding it?

Reply:
Sort of, but not really. We'd have to scan through all the mods, then split them up. I kinda wanted to re-use the same export function as S3 as for the table dump/restore functions, so this was the middle ground.

Reply:
Databases typically have indexing capabilities that can handle this kind of operation very quickly. My guess is it's somewhere in pynamodb as well.

Reply:
DynamoDB isn't really a traditional database, so that's not quite how the indexing works. So you have to scan all the entries and find the uniqueness client side.